pkgs/development/python-modules/py-slvs/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, blender, python3Packages, pkgs, fetchurl }:
```

```suggestion
{
  lib,
  buildPythonPackage,
  fetchPypi,
  swig,
  cmake,
  ninja,
  setuptools,
  scikit-build,
}:
```

Since this file if being called with `callPackage` from inside `pkgs/top-level/python-packages.nix` this means that both python and non-python packages will be populated automatically in this top attrset.

By doing this you can remove the explicit references to `python3Packages` and `pkgs`.